worker/lib/workers/qemu.ts

```diff
@@ -170,6 +181,47 @@ class QemuWorker extends EventEmitter implements Leviathan.Worker {
 		this.activeFlash = undefined;
 	}
 
+	private async findUEFIFirmware(architecture: string)
+		: Promise<undefined | {code: string, vars: string}> {
+		const searchPaths: { [arch: string]: {code: string, vars: string}[] } = {
```

Comment:
@jakogut you already created a typed called Firmware over in worker.d.ts, is there any reason you chose to not use them here? This should work... 

```
Promise<undefined | firmware>
```